From 5e9d397122b34d4b8fa00b69e22c7d5c01cd0347 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 31 Oct 2018 18:02:35 -0700
Subject: [PATCH 5/7] DO-NOT-UPSTREAM: Disable SAMPLE_VFS

It's broken for me and I don't care enough to look into it.

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 samples/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/samples/Kconfig b/samples/Kconfig
index dc4eb5355fad..c83741b9d87e 100644
--- a/samples/Kconfig
+++ b/samples/Kconfig
@@ -149,6 +149,7 @@ config SAMPLE_VFIO_MDEV_MBOCHS
 
 config SAMPLE_VFS
 	bool "Build example programs that use new VFS system calls"
+	depends on BROKEN
 	help
 	  Build example userspace programs that use new VFS system calls such
 	  as mount API and statx().  Note that this is restricted to the x86
-- 
2.19.1

